spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -108,6 +108,12 @@
         .add("args", "", "spark commandline args")
         .add("zeppelin.spark.printREPLOutput", "true",
           "Print REPL output")
+        .add("spark.mahout", "false", "add Apache Mahout Support Support for Spark")
+        .add("spark.kryo.registrator",
+          "org.apache.mahout.sparkbindings.io.MahoutKryoRegistrator", "")
```

Comment:
Will there be no side effect of setting `org.apache.mahout.sparkbindings.io.MahoutKryoRegistrator` as a default value of `spark.kryo.registrator`, especially when `spark.mahout` is `false`?